### DISCUSSION TITLE
1:Web Button (Script) 2:RFSend done feedback

### DISCUSSION THREAD

**Discussion by martin072** [arendst/Tasmota#13681] [Q&A] [answered]
2 question actually..

1: 
I noticed in version 10.0 (release) when I create a web button in a script it displays garbled text (see attachment). Tried two different browser, both the same. Does not matter if you write "1", or "ON" or "OFF" 

Web Button image:

<img width="428" alt="Screenshot 2021-11-15 at 13 05 46" src="https://user-images.githubusercontent.com/24524506/141779989-85e8ae61-8994-4791-9192-8de4362a6917.png">

2: Would it be possible to include the send RF Code in the feedback? 
i.e from {"RFSend":"Done"} --> {"RFSend":"Done", "RFCode": 105588}?
Reason for this is to track when certain codes have been send either by tasmota or the physical remote

**Comment by Jason2866** [comment, marked as answer]
Question 1:
@gemu2015 can you help?
Question 2:
This message ONLY is sent when Tasmota has sent a RF code. When a RF Remote has done you will see a RF Receive message in Tasmota. Adding this info has high chance to break current HomeAutomation solutions.

**Reply by gemu2015** [reply]
sorry can not reproduce with latest dev 10.0.0.3
since nothing has changed in this interface recently i can not explain your bug.

**Reply by martin072** [reply]
the Button Was my mistake, forgot to initialise the variable. Doen't like that apparently..
 
As for the RFSend, understand that. Thanks for your quick reply's
